src/bc.F

```diff
@@ -3656,10 +2947,26 @@ subroutine bcs2_GPU(s)
 
       endif
       
-      !$acc end data
-
       if(timestats.ge.1) time_bcs2=time_bcs2+mytime()
 
-      end subroutine bcs2_GPU
+      end subroutine bcs2
+
+      subroutine SetGPUParams(openacc,device)
+
+         logical, intent(inout) :: openacc
+         logical, optional :: device
+
+      if(present(device)) then
+         if(device) then
+            openacc = .true.
+         else
+            openacc = .false.
+         endif
+      else
+         openacc = .true.
```

Comment:
This is just a preference question: is `.false.` or `.true.` better as the default value here?

Reply:
This is probably a longer discussion that we can have in person versus through github.

Reply:
After the discussion with @johnmauff , it seems reasonable to set `.true.` as the default value and let the user specify which subroutines they want to run on CPU instead.